sdk/include/opentelemetry/sdk/common/atomic_unique_ptr.h

```diff
@@ -61,7 +61,7 @@ class AtomicUniquePtr
 
   /**
    * Atomically swap the pointer with another.
-   * @param ptr the pointer to swap with
+   * @param other the pointer to swap with
```

Comment:
Nit: To maintain consistency, start the param definition with a capital letter. 
`the pointer ...` &rarr; `The pointer ...` 

Also, in other places.